kitsune/questions/views.py

```diff
@@ -244,7 +244,8 @@ def question_list(request, product_slug):
     question_qs = question_qs.filter(locale_query)
 
     # Set the order.
-    order_by = ORDER_BY[order][0]
+    # Set a default value if a user requested a non existing order parameter
+    order_by = ORDER_BY.get(order, 'updated')
```

Comment:
This needs to be something like `ORDER_BY.get(order, ['updated'])[0]` otherwise it breaks when `order` exists in the dict.